tiledb/sm/c_api/tiledb.cc

```diff
@@ -490,7 +490,7 @@ int tiledb_config_get(
   return TILEDB_OK;
 }
 
-int tiledb_config_load_from_file(
+int32_t tiledb_config_load_from_file(
     tiledb_config_t* config, const char* filename, tiledb_error_t** error) {
   if (sanity_check(config, error) == TILEDB_ERR)
```

Comment:
Should the sanity checks be converted from int return values? I realize your focus is on the public API but if we are saying TILEDB_ERR/OK/OOM is a int32_t should these be modified for consistency?

Reply:
Yes they should be `int32_t` as well for consistency, I will modify.